plugin/src/main/java/io/deephaven/plugin/type/ObjectType.java

```diff
@@ -63,6 +89,7 @@ interface Exporter {
          * @param forceNew if a new reference should be created
          * @return the reference
          */
+        @Deprecated
```

Comment:
should either remove these methods outright, or document that the params will be disregarded. I'm leaning towards remove.

Also should probably remove the interface entirely, as it no longer has an implementation from the grpc impl, but just as a simpler way to handle non-bidi widgets.

Reply:
Reviewers, thoughts? I've documented it for now, but removal will be even easier.